site/guide/work-with-content-blocks.qmd

```diff
@@ -50,6 +50,9 @@ Test-driven block
 After you have completed these steps, the new content block becomes a part of your model documentation.
 
 <!--- TO DO We might need to revisit terminology: we talk about "block", "section", and "module", but they all seem to be names for very similar things.--->
+### Add mathematical formulas
+
+While editing a content block, you can insert math equations using the editor. Its interface is accessed by clicking the square-root icon within the toolbar. You can use the interface to type out the equation, or paste in a LaTeX formula. See [the editor's docs](https://ckeditor.com/docs/ckeditor5/latest/features/math-equations.html) for detailed instructions.
```

Comment:
FYI, links to external docs can break and I don't think external links get checked when we generate docs. Seems useful here, but I would use links like these sparingly, as they invariably all break eventually.